testgrid/config.yaml

```diff
@@ -7603,6 +7607,9 @@ dashboards:
   - name: "Conformance acs-engine on Azure"
     description: Runs conformance tests on a Kubernetes cluster provided by acs-engine (https://github.com/Azure/acs-engine) on Azure cloud"
     test_group_name: ci-kubernetes-e2e-win-1-13
+  - name: "Conformance kubo on vSphere"
+    description: Runs conformance tests on a Kubernetes cluster provided by kubo (https://github.com/cloudfoundry-incubator/kubo-release) on vSphere"
```

Comment:
can you provide more details about the kubernetes version under test? is it a CI build? which branch or release? This is a windows cluster?

Reply:
We are building master daily using Concourse CI, and deploying a cluster with 3 Windows nodes and 1 (tainted) Linux node. We taint the Linux node just so that the test workloads don't end up on it. This file has all our test setup: https://github.com/pivotal-k8s/pksw-ci/blob/0c4bf87728920493d15172bb2a9fc4e114e16a5f/tasks/run-e2e-tests.sh

Reply:
thanks! sorry this slipped my inbox

Reply:
non-blocking: can we please specify what kubernetes version / build is tested in the description?

Knowing what build is tested is extremely helpful. Eventually I plan to request users of the conformance upload script to start supplying the kubernetes commit under test, our own CI records this generally.

Additionally we maybe (?) should not call these windows jobs conformance yet since windows conformance is not defined currently.

Reply:
@BenTheElder Sounds good; I'll redo the description.
Is there are place to record the commit currently when using the script? Our CI would have that information when it's packaging/uploading the results so it's easy to fill it in somewhere.

Do you prefer the term "verification" over conformance for now? I think that's what I've heard the current windows tests referred to as.

Reply:
verification sounds fine, punting to @spiffxp though :thinking: 

Adding a `"job-version": "kubernetes-sha"` to the finished.json would cause it to be picked up IIRC, but we don't have a flag in the tool yet (should be a small change.

Reply:
thanks :-)